resources/templates/account.html

```diff
@@ -53,31 +53,19 @@
     <div class="tooltip">
       <ul>
         <li>
-          Raiden accounts are Ethereum accounts defined by a pair of keys, a
-          private key and public key.
+          Download you keystore file now. You can restore your Raiden account
+          with this file and the password via Metamask.
         </li>
         <li>
-          Every private key/address pair is encoded in a keyfile. Keyfiles are
-          JSON text files which you can open and view in any text editor.
+          Send 0.125 ETH to your Raiden account. 0.075 ETH are used as safety buffer
+          to operate 3 channels. 0.05 ETH are needed for RDN and DAI tokens.
         </li>
-        <li>You need to download and store this keyfile somewhere safe.</li>
         <li>
-          The critical component of the keyfile, your account's private key, is
-          always encrypted, and it is encrypted with the password you entered
-          when you created the account.
+          You can withdraw those 0.125 ETH if not spent in a transaction or
+          as transaction fees.
```

Comment:
What do you mean by `withdraw as transaction fees`

Reply:
`          You can withdraw those 0.125 ETH minus the amount you will have spent in a 
          transaction or as transaction fees.`

besser?

Reply:
perfekt. Jetzt verstehe ich was du meinst